src/pages/managerForm.js

```diff
@@ -1,66 +1,387 @@
-                        onClickProp={categoryChoice}
+                        onClick={categoryChoice}
                         text="Software Engineering"
                     />
-
                     <CustomizableButton
                         customClass={styles.disableButton}
-                        onClickProp={categoryChoice}
+                        onClick={categoryChoice}
                         text="Product Design"
                     />
-
                     <Image
                         src='/arrow-right-line.png'
                         width={25}
                         height={25}
                         alt='arrow pointing to the right'
                     />
                 </div>
+            </div>
+
+            <form 
+                onSubmit={submitCommunity}
+                action="/send-data-here" 
+                method="POST"
+            >
+                <container className={styles.communityLayout}>
+                    <div className={styles.layoutRow}>
+                        <label 
+                            htmlFor='community_name'
+                        >Community Name
+                        </label>
+                        <input 
+                            type='text' 
+                            name='community_name' 
+                            id='community_name' 
+                            value={formData.community_name}
+                            onChange={handleChange}
+                            required 
+                            className={styles.inputStyle}
+                        />
+                    </div>
+                    <div className={styles.layoutRow}>
+                        <label 
+                            htmlFor='community_manager'
+                        >Community Manager
+                        </label>
+                        <input 
+                            type='text' name='community_manager' id='community_manager' value={formData.community_manager}
+                            onChange={handleChange}
+                            required className={styles.inputStyle}
+                        />
+                    </div>
+                    <div className={styles.layoutRow}>
+                        <label 
+                            htmlFor='community_date'
+                        >Date Established
+                        </label>
+                        <input 
+                            type='text' 
+                            name='community_date' 
+                            id='date' 
+                            value={formData.community_date} onChange={handleChange}
+                            className={styles.inputStyle} 
+                        />
+                    </div >
+                    <div className={styles.layoutRow}>
+                        <label  
+                            htmlFor='community_location'
+                        >Location
```

Comment:
Do we have a "Location" field in the DB?

If not, remove from form.

If yes, I think we should add a description under the field that reads: _If you host in-person events, type your City/State and/or Country. If not, type 'Online'_

Reply:
We don't, I will remove.